js/app/image-lazy-loader.js

```diff
@@ -0,0 +1,46 @@
+var ImageObserver = (function() {
+    var observer;
+
+    function createObserver() {
+
+        // small polyfill for Microsoft Edge 15 isIntersecting property
+        // see https://github.com/WICG/IntersectionObserver/issues/211#issuecomment-309144669
+        if ("IntersectionObserver" in window &&
```

Comment:
IntersectionObservers are not supported on Safari on iOS or desktop. What's the fallback strategy for those browsers?